examples/listbox/listbox-grouped.html

```diff
@@ -46,21 +46,21 @@ <h2 id="ex_label">Example</h2>
           <span id="ss_elem" class="listbox-label">Choose your animal sidekick</span>
           <div id="ss_elem_list" tabindex="0" role="listbox" aria-labelledby="ss_elem">
             <ul role="group" aria-labelledby="cat1">
-              <li role="presentation" id="cat1">Land</li>
+              <li aria-hidden="true" id="cat1">Land</li>
```

Comment:
Using aria-hidden to manage screen reader verbosity related to elements referenced by aria-labelledby has been contraversial. The task force has not supported doing so. It is a technique that can have severe negative consequences. In this particular case, it might be innocuous, but is addressing a symptom of a larger problem rather than the root cause, doing so  with a risky approach.